src/panel/category_panel.js

```diff
@@ -87,8 +87,6 @@ export default class CategoryPanel {
 
     this.addCategoryMarkers();
     fire('save_location');
-
-    document.querySelector('.service_panel').classList.remove('service_panel--active');
```

Comment:
When the service panel isn't shown, it isn't in the DOM at all, so we don't need to control it explicitely like this anymore => one less ugly dependency.